guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/mutations/index.md

```diff
@@ -89,6 +89,41 @@ Grab the second string, lowercase and turn it into an array; then make sure _eve
 *   <a href='http://forum.freecodecamp.com/t/javascript-string-prototype-split/15944' target='_blank' rel='nofollow'>Array.split()</a>
 *   <a href='http://forum.freecodecamp.com/t/javascript-array-prototype-every/14287' target='_blank' rel='nofollow'>Array.every()</a>
 
+## ![:rotating_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/rotating_light.png?v=3) Advanced Code Solution:
+
+**Procedural**
+
+```js
+
+    function mutation(arr) {
+      let re = new RegExp("(?=.*" + arr[1].toLowerCase().split("").join(')(?=.*') + ")","ig");
+      return re.test(arr[0]);
+    }
+
+     console.log(mutation(["hello", "Hello"]));
+
+```
+     
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/@cybcova/Mutations-New-Regex-Solution' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
Please remove the repl.it link.  We no longer allow linking to resources like this which we do not control.  Also only leave one blank line between the triple backtick code fence and the `### Code Explanation` line.
```suggestion
```